spacepy/irbempy/NOTES.md

```diff
@@ -1,28 +1,43 @@
 # Notes on how to update the revision of IRBEMlib
+First you'll need to check out the IRBEM git repository from
+https://github.com/PRBEM/IRBEM
+Some required files get built by make, so in the IRBEM repo:
+```
+make OS=linux64 ENV=gnu64 all
+```
 
-### convert .../source/drift_bounce_orbit.f to ASCII
-iconv -f ISO-8859-1 -t ASCII -c drift_bounce_orbit.f > drift_bounce_orbit_new.f
-mv drift_bounce_orbit_new.f drift_bounce_orbit.f
+The `compare_irbem.py` script in `spacepy/developer/scripts` will list the
+files that differ between the IRBEM repo and the current spacepy version.
+It will also give line numbers with unicode characters that need replacing.
 
-### replace myOwnMagField.f with Dungey model
-cp [previousRev]/source/myOwnMagField.f [newRev]/source/myOwnMagField.f
+### If necessary, update myOwnMagField.f
+SpacePy uses the `myOwnMagneticField.f` to implement the "Dungey" model.
+In this case that's a centered dipole plus uniform field.
+In the unlikely event that there's a required change to the calling syntax
+or similar, this file will need updating. Otherwise, it's expected to be
+different since the file in the IRBEM directory is a stub.
 
 ### fix comments; unicode
 It seems that some comment lines confuse f2py.
-E.g., line 690 (in rev546) "      REAL*8 Bmirror  ! particle's mirror field strength"
-The apostrophe breaks the compile, so it needs to be removed.
-Lines 331 and 3083 (in rev616) have unicode degree symbols that should be removed.
-There are a lot of wanrings about lines with an exclamation as comment marker - for now we ignore these.
```

Comment:
Did this go away for a reason? Do we no longer have the warnings now that we're compiling with different flags?

Reply:
I fixed some of these upstream, and any remaining get manually fixed on updating the version in spacepy. I honestly don't remember what the balance of those is, but it's way less work now than it used to be...